docs/sources/writing-guide/topic-types/tutorial/index.md

```diff
@@ -96,6 +96,49 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
 
 If getting access to the tutorial data is complex, include the instructions in the steps of the tutorial. If getting access to the data is straightforward, include it in the "Before you begin" section.
 
+## Where to locate your tutorial
+
+Sometimes it makes sense to publish your tutorial directly within the documentation for your product or feature. If you want higher visibility for your tutorial, you can add it to the main Grafana [Tutorials](/tutorials) page. Even though you're displaying your tutorial on the Tutorials page, you need to add your tutorial source to your team's repo to make it easy for team members to review the content. 
+
+Follow the steps below to hide the tutorial from your documentation's table of contents and display it on the Tutorials page.
+
+> Note: This procedure is for writers who have permissions to update the Grafana website repo.
+
+To publish your tutorial to the main Tutorials page:
+
+1. In your documentation repo, create a folder under `sources` called `tutorials` and add your tutorial to this folder.
+
+2. To hide your tutorial from your documentation table of contents, add the following to the metadata of your `sources/_index.md` file (replace your repo for `grafana-cloud`):
+
+    ```
+    cascade:
```

Comment:
For this and all the subsequent instructions that refer to YAML, we actually want the user to integrate the snippet with their existing content.

If they already have a `cascade` key, they should not introduce another. Similarly we do not want to introduce a new `manual_mounts` object to the website `config.yaml` but just add a new object to the existing sequence.

In the past I've written something like:

Add the following to the `$.cascade` field in your YAML metadata, creating the `cascade` field if it does not exist:

```yaml
- _target:
    path: /docs/grafana-cloud/tutorials/**
  _build:
    list: false
    render: false
- _target:
    path: /docs/grafana-cloud/**
```